scopes/react/react/apps/web/compute-results.ts

```diff
@@ -1,19 +1,23 @@
 import { BundlerResult } from '@teambit/bundler';
 import { ReactAppBuildResult } from './react-build-result';
 
-type resultsOptions = {
+type ResultsOptions = {
   publicDir: string;
   ssrPublicDir?: string;
 };
 export async function computeResults(
   results: BundlerResult[],
-  { publicDir, ssrPublicDir }: resultsOptions
+  { publicDir, ssrPublicDir }: ResultsOptions
 ): Promise<ReactAppBuildResult> {
   const result = results[0];
 
   return {
     publicDir,
     ssrPublicDir,
```

Comment:
can't we delete these now once we move it to metadata?
or do you keep it for backward compatibility? (if yes, please add a comment above that it is the case and it should be remove at some point)

Reply:
Yes. It's for backward compatibility. I'll add comments.